server/migrations/20181115171611-update-complaint-model.js

```diff
@@ -0,0 +1,16 @@
+export default {
+  up: (queryInterface, Sequelize) => {
+    queryInterface.addColumn(
+      'Complaints',
+      'adminAction',
+      {
+        type: Sequelize.TEXT,
```

Comment:
@ProfJigsaw Why are we using TEXT here instead of STRING dataType?

Reply:
The admins response could be anything from a simple `Thank you for your concern. We are processing your request` to a full text response dispelling a raised complaint.